spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -292,17 +309,26 @@ def create!(csr: nil, bundle_id: nil)
           mac = MAC_CERTIFICATE_TYPE_IDS.include?(type)
 
           # look up the app_id by the bundle_id
-          if bundle_id
+          if bundle_id && !bundle_id.start_with?('merchant.')
             app = Spaceship::Portal::App.set_client(client).find(bundle_id)
             raise "Could not find app with bundle id '#{bundle_id}'" unless app
             app_id = app.app_id
+          else
+            merchant = Spaceship::Portal.merchant.find(bundle_id)
+            raise "Could not find merchant with bundle id '#{bundle_id}" unless merchant
+            merchant_id = merchant.merchant_id
           end
 
           # ensure csr is a OpenSSL::X509::Request
           csr = OpenSSL::X509::Request.new(csr) if csr.kind_of?(String)
 
           # if this succeeds, we need to save the .cer and the private key in keychain access or wherever they go in linux
-          response = client.create_certificate!(type, csr.to_pem, app_id, mac)
+          if type == '4APLUP237T'
```

Comment:
What is this magic number? Extract to a variable so it can be documented via the var name maybe?

Reply:
@janpio This magic number is defined by Apple. It's also used in other places in the `fastlane` repo. https://github.com/fastlane/fastlane/blob/master/spaceship/lib/spaceship/portal/certificate.rb#L165

Reply:
Let's make this a bit more explicit here:
```suggestion
          if type == '4APLUP237T' # = ApplePay
```

Better would be if we could just read this from a constant by key `ApplePay` here of course, but we don't seem to have this in the code base in that direction, only from the magic number _to_ Apple Pay.